### PR TITLE
[Ruby] Bump up ms_rest & ms_rest_azure version to 0.6.0

### PR DESCRIPTION
**Pull request by vishrutshah**

### CODE REVIEW

**Comment by vishrutshah** [comment]
@veronicagg & @sarangan12 Feel free to review the PR when you get a chance. 

@dsgouda FYI Once this is been reviewed & merged, we'll ship the new runtime for you to use as `~> 0.6.0`

**Comment by veronicagg** [comment]
Rest of the code changes look good to me, thanks!

**Comment by vishrutshah** [comment]
Of course. Thanks for catching that. Updated the usage instructions for generator.

File: src/client/Ruby/ms-rest-azure/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+##2016.10.04 ms_rest version 0.6.0
```
Comment (veronicagg) [inline src/client/Ruby/ms-rest-azure/CHANGELOG.md]: oh, just realized that the top comment should say ms_rest_azure intead of ms_rest for this changelog entry

Comment (vishrutshah) [inline src/client/Ruby/ms-rest-azure/CHANGELOG.md]: sure. my bad copy paste :(

**Comment by vishrutshah** [comment]
Thanks for the review!!

**Pull request merged**